openpype/modules/ftrack/event_handlers_user/action_create_folders.py

```diff
@@ -196,6 +228,27 @@ def launch(self, session, entities, event):
             if not os.path.exists(path):
                 os.makedirs(path)
 
+        self.log.info("Creating local template folders...")
+
+        if from_local:
```

Comment:
Not sure if this belongs here. The logic is already in different action (`CreateProjectFolders`).